src/main/java/org/kohsuke/github/Preview.java

```diff
@@ -15,4 +15,15 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
 public @interface Preview {
+
+    /**
+     * An optional field defining what API media types must be set inorder to support the usage of this annotations
+     * target.
+     * <p>
+     * This value should be set using the existing constants defined in {@link Previews}
+     *
+     * @return The API preview media type.
+     */
+    public String[] value() default {};
+
```

Comment:
Very nice.  I'd almost prefer it to not be optional but I see why it needs to be.